sidebar.cpp

```diff
@@ -27,15 +27,17 @@ PlanetInspector::PlanetInspector(Planet *planet)
 
         physical = TextWidget(&header, "Physical Properties", true);
         distance = TextWidget(&body, "Distance from sun: " + std::to_string(int(planet->distanceFromSun / 80)) + " AU", false);
-        radius = TextWidget(&body, "Radius: " + std::to_string(int(planet->radius)) + " miles", false);
```

Comment:
This change has political connotations (lol). I took the liberty to change from miles to the SI because calculating any physical quantity outside of SI is really painful.

If it's unbearable, we can make a conversion m->miles when we update the TextWidgets :)

Reply:
Yeah - I agree. The only reason it was miles originally was because they were the units most things I looked up were in.